external/os-autoinst-common/tools/update-deps

```diff
@@ -77,7 +77,7 @@ RUN zypper in -y -C \\
 $dep   && zypper clean
 EOM
     $docker =~ s/($begin\n)(.*)($end\n)/$1$run$3/s;
-    path($dockerfile)->spurt($docker);
+    path($dockerfile)->spew($docker);
```

Comment:
Don't forget to do a `git subrepo push external/os-autoinst-common` after the merge

Reply:
https://github.com/os-autoinst/os-autoinst-common/pull/27

Reply:
I wrote `git subrepo push external/os-autoinst-common` for a reason :)

Reply:
what's wrong with my extra PR?

Reply:
https://github.com/os-autoinst/os-autoinst-common/pull/27#issuecomment-1717424808

Reply:
ok, marked this PR as draft. I suggest to have https://github.com/os-autoinst/os-autoinst-common/pull/27 merged first and then pull in and rebase here